### PR TITLE
Request/response overview, json api glossary

### PR DESCRIPTION
**Pull request by stefanorosanelli**
Solves #14

### CODE REVIEW

File: source/endpoints/request_response.rst [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+This is the classical response in case of resources list. For a single resource response is very similar, but somehow reduced.
+
+At the root level we have:
+
+ * ``"data"`` key will contain an array of resources, or a single resource when a single resource is requested like in **GET /documents/2**
+ * ``"meta"`` will contain metadata of single resources or of resource lists, mainly pagination info for lists
+ * ``"links"`` key appears in more than one place and will contain
+
+    - ``"self"`` (always present) with a link pointing to the resource containing it; it can be an array of resources, a single resource ora a relationship
```
Comment (batopa) [inline source/endpoints/request_response.rst]: typo `ora`

File: source/endpoints/request_response.rst [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+ * ``"attributes"`` is for resource attributes, these are generally properties that a client application can modify, once permissions are granted, and contains
+
+    - **stantard core** properties like ``"title"``, ``"description"``, ``"lang"`` or ``"body"`` available for all object types
+    - **custom** properties that belong to the current resource or object type only - ``"custom_field"`` in this example
+    - **special core** properties available for all object types, with some specific meaning and usage:
+
+        * ``"status"`` may have only ``on``, ``draft`` or ``off`` as values; it representes an editorial status, depending on client application settings objects with status ``draft`` and ``off`` may not be visible
+        * ``"uname"`` is the object unique name, a modifiable unique identifier of every object inside a project, represented by an URL-friendly string that may be derived from the ``"title"`` or from other properties; upon creation a ``uname`` is automatically created and when a client applications request to create or modifiy this property the actual value may be changed by the system to avoid conflicts with an existing identical ``uname``
```
Comment (batopa) [inline source/endpoints/request_response.rst]: maybe `application`, not plural?

File: source/endpoints/request_response.rst (hunk 1 above)
Comment (stefanorosanelli) [reply source/endpoints/request_response.rst]: corrected

File: source/endpoints/request_response.rst (hunk 2 above)
Comment (stefanorosanelli) [reply source/endpoints/request_response.rst]: yes corrected this way: `a client application requests...`

**Pull request merged**